level-ws.js

```diff
@@ -69,62 +73,58 @@ inherits(WriteStream, Writable)
 
 WriteStream.prototype._write = function write (d, enc, next) {
   var self = this
+  // TODO use self.destroyed
   if (self._destroyed) return
 
+  // TODO remove, no longer needed
   if (!self._db.isOpen()) {
     return self._db.once('ready', function () {
       write.call(self, d, enc, next)
     })
   }
 
-  if (self._options.maxBufferLength &&
-      self._buffer.length > self._options.maxBufferLength) {
```

Comment:
This commit temporarily removes support for `maxBufferLength`. Focus was to remove race conditions first.